tcms/rpc/api/testcase.py

```diff
@@ -279,8 +283,24 @@ def filter(query=None):  # pylint: disable=redefined-builtin
     if query is None:
         query = {}
 
-    return list(
-        TestCase.objects.filter(**query)
+    # Cannot use `setup_duration` and `testing_duration` as aliases because Django
+    # doesn't allow annotating querysets with aliases which match existing fieldnames.
+    # https://code.djangoproject.com/ticket/11256
+    # https://github.com/django/django/blob/main/django/db/models/query.py#L1154-L1157
+
+    # we're adding the suffix `_ms` to the aliases because they are
+    # `Cast`ed to their millisecond values
```

Comment:
The actual values appear to be in microseconds though. A duration of 4 seconds is returned with value `4000000.0` by the query. 

IMO let's return these values in seconds (suffix _sec) by dividing by 10**6.

Reply:
Ah, yes. I wasn't being attentive. They are microseconds 🤦🏾.